code/requirements.txt

```diff
@@ -1,3 +1,4 @@
 cosmotech-run-orchestrator
 cosmotech-api~=3.1.0
 azure-storage-blob
+CosmoTech-Acceleration-Library
```

Comment:
Could we specify at least the major version to use, to avoid unexpected surprises with new versions of the package? :smile: